.github/workflows/cicd.yml

```
@@ -19,10 +19,53 @@ jobs:
         with:
           distribution: 'temurin'
           java-version: '17'
+      - name: Gradle 테스트 및 JaCoCo 리포트 생성
+        run: ./gradlew clean test jacocoTestReport
 
-      - name: 테스트 실행 스킵 알림
-        run: |
-          echo "::notice::테스트 코드는 실행하지 않았지만, 빌드는 정상적으로 완료했습니다."
+      - name: JaCoCo 리포트 업로드
+        uses: actions/upload-artifact@v2
```

actionlint (1.7.4)

the runner of "actions/upload-artifact@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)